faq/running.inc

```diff
@@ -1634,6 +1634,32 @@ $a[] = "See these two FAQ categories:
 
 /////////////////////////////////////////////////////////////////////////
 
+$q[] = "What are the libfabric (OFI) components in Open MPI?";
+
+$anchor[] = "mpi-ofi-support";
+
+$a[] = "
+Open MPI has two main components for Libfabric (a.k.a OFI) communications:
+
+*OFI MTL:* For two sided tag matching MPI communication. Included since v1.10.
+Works with providers that support:  EP type: FI_EP_RDM; Capabilities: FI_TAGGED ; Ordering: FI_ORDER_SAS
+
+*OFI BTL:* For one sided communications. Included since v4.0.
+Works with providers that support:  EP type: FI_EP_RDM; Capabilities: FI_RMA and FI_ATOMIC; Op flags: FI_DELIVERY_COMPLETE;
+This BTL is expected to be used with OSC RDMA.
+
+See the per provider man page to understand which provider will work for each of the components.  Some providers may require to be paired with the utility provider, for example, verbs provider needs to be paired with utility provider ofi_rxm to provide reliable datagram end points (verbs;ofi_rxm)
+
+Both components have mca para meters to specify the OFI provider to be included/excluded in the selection process. In addition each component has specific parameters for each one (see ompi_info --param <framework> <component>).
```

Comment:
Word smything:

See each Lifabric provider man page (e.g., fi_sockets(7)) to understand which provider will work for each of the above-listed Open MPI components.  Some providers may require with the Libfabric utility provider; for example, the verbs provider needs to be paired with utility provider `ofi_rxm` to provide reliable datagram endpoint support (`verbs;ofi_rxm`).

Both components have MCA parameters to specify the Libfabric provider(s) that will be included/excluded in the selection process. For example:

$ mpirun --mca ofi_mtl_provider_include ...PUT_AN_EXAMPLE_HERE...
# JMS Not sure I got the MCA param name correct -- please put in the right name.

In addition, each component has specific parameters for each one; see `ompi_info --param <framework> <component>` for a full list.